dap_prinz_green_jobs/pipeline/green_measures/occupations/README.md

```diff
@@ -3,12 +3,30 @@
 The script
 
 ```
-python dap_prinz_green_jobs/pipeline/green_measures/occupations/occupation_measures_utils.py
+dap_prinz_green_jobs/pipeline/green_measures/occupations/occupation_measures_utils.py
 ```
 
 contains functions needed to see whether occupations are green or not.
 
+The class in
+
+```
+dap_prinz_green_jobs/pipeline/green_measures/occupations/soc_map.py
+```
+
+can be used to map job title(s) to SOC codes, for example
+
+```
+from dap_prinz_green_jobs.pipeline.green_measures.occupations.soc_map import SOCMapper
+
+soc_mapper = SOCMapper()
+soc_mapper.load()
```

Comment:
wonder if we should have a .load() step for the GreenMeasures class given how long it takes to instantiate